src/components/Garden/ProposalDetail/ProposalActions.js

```diff
@@ -41,13 +41,16 @@ function ProposalActions({
       return null
     }
     if (currentConviction.gte(threshold)) {
-      return 'execute'
+      if (proposal.statusData.open) {
+        return 'execute'
+      }
+      return null
```

Comment:
Probably we don't want to hide actions when the proposal is challenged or disputed.

Let's say i staked and the proposal got challenged and it reached sufficient conviction but i want to update my support. 

```suggestion
```

Reply:
good catch!